<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrillingMachine</class>
 <widget class="QWidget" name="DrillingMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>633</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox{
background-color: rgb(211, 215, 207);
border-color: rgb(239, 41, 41);
selection-color: rgb(153, 255, 0);
margin-top:5px;
border: 1px solid gray;
border-radius: 4px;
padding-left:6px;
}
background-color: rgb(196, 160, 0);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Coordinatus</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="baseSize">
           <size>
            <width>280</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(114, 159, 207);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Model statistic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_4{
background-color: rgb(211, 215, 207);
border-color: rgb(239, 41, 41);
selection-color: rgb(153, 255, 0);
margin-top:0px;
border: 1px solid gray;
border-radius: 8px;
padding-left:6px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>280</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_3{
background-color: rgb(211, 215, 207);
border-color: rgb(239, 41, 41);
selection-color: rgb(153, 255, 0);
margin-top:0px;
border: 1px solid gray;
border-radius: 8px;
padding-left:6px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Autolevel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_5{
background-color: rgb(211, 215, 207);
border-color: rgb(239, 41, 41);
selection-color: rgb(153, 255, 0);
margin-top:0px;
border: 1px solid gray;
border-radius: 8px;
padding-left:6px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
